libs/ssl-config/src/main/java/org/elasticsearch/common/ssl/SslConfigurationLoader.java

```diff
@@ -68,6 +68,7 @@
 
     static final List<String> DEFAULT_PROTOCOLS = List.of("TLSv1.3", "TLSv1.2", "TLSv1.1");
 
+    // @formatter:off
```

Comment:
This seems like an important tool we should document in our CONTRIBUTING.md? (separate from this PR)

Reply:
I can add it to `CONTRIBUTING.md`, I should have adding it in the last PR to be honest.